pkg/controller/account/account_controller.go

```diff
@@ -41,6 +41,9 @@ const (
 	awsSecretName           = "aws-config"
 	awsAMI                  = "ami-000db10762d0c4c05"
 	awsInstanceType         = "t2.micro"
+	AccountCreating         = "Creating"
```

Comment:
I think we'll need comments for all of these to keep linting happy?

Reply:
added comments